src/Worker.Extensions.DurableTask/AssemblyInfo.cs

```diff
@@ -4,4 +4,4 @@
 using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
 // TODO: Find a way to generate this dynamically at build-time
-[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.DurableTask", "2.7.*")]
+[assembly: ExtensionInformation("Microsoft.Azure.WebJobs.Extensions.DurableTask", "2.8.*")]
```

Comment:
did you modify this on purpose? Seems pretty low-level

Reply:
Yes, I changed this based on the last item on the PR checklist. I figured since we're moving to a new replay schema, it warranted a new minor version, but if I'm wrong I can revert this back to what it was

Reply:
I see. That new checklist item is fairly new, so I'm personally unsure how exactly to interpret it, but I'm almost certain we're not supposed to be updating it here :-)

@cgillum: can you please clarify that final bullet-point in the PR checklist?

Reply:
The idea is that if you are incrementing the extension version number then you must also increment the version number in this file to ensure they always match each other. For example, this update makes sense IFF we're also updating the version number in WebJobs.Extensions.DurableTask.csproj.

Reply:
Thanks for clarifying @cgillum.

@hossam-nasr: I don't think we should update the extension version number in this PR, we usually do that during our release process, so let's please revert this change for now. During release ,we'll make sure this file is properly updated if necessary :) . Thanks!

Reply:
Yeah that makes sense, sorry for the confusion and thanks for the clarification. I reverted this line